src/anndata/tests/test_concatenate.py

```diff
@@ -1452,12 +1465,13 @@ def test_concat_null_X():
 
 # https://github.com/scverse/ehrapy/issues/151#issuecomment-1016753744
 @pytest.mark.parametrize("sparse_indexer_type", [np.int64, np.int32])
-def test_concat_X_dtype(array_type, sparse_indexer_type):
+def test_concat_X_dtype(cpu_array_type, sparse_indexer_type):
     adatas_orig = {
-        k: AnnData(array_type(np.ones((20, 10), dtype=np.int8))) for k in list("abc")
+        k: AnnData(cpu_array_type(np.ones((20, 10), dtype=np.int8)))
+        for k in list("abc")
     }
     for adata in adatas_orig.values():
-        adata.raw = AnnData(array_type(np.ones((20, 30), dtype=np.float64)))
+        adata.raw = AnnData(cpu_array_type(np.ones((20, 30), dtype=np.float64)))
```

Comment:
Can we just change the dtype from `uint8` here?  Or is there a reason to keep this?  I think that this is what is causing GPU test failures